src/components/Login/Login.js

```diff
@@ -48,7 +48,7 @@ class Login extends Component {
     const {
       handleSubmit,
       submitting,
-      authError,
+      authErrors = false,
```

Comment:
Are you sure that default value for an array should be false? Please use static field defaultProps.
`static defaultProps = {
   authErrors: [],
}`